container_images/daisy-builder/main.sh

```diff
@@ -81,6 +83,24 @@ function generate_build_workflow() {
   echo -e "$config" > "$WF"
 }
 
+tag_commit() {
+  LATEST_VERSION=$(generate_new_version)
+  if [[ "$LATEST_VERSION" != "$VERSION" ]]; then
```

Comment:
...also returned here by the function, and then compared. won't these always be equal?

Reply:
fixed